_data/developer.yml

```diff
@@ -200,6 +200,12 @@ websites:
       img: newrelic.png
       tfa: No
 
+    - name: Nexmo
+      url: https://www.nexmo.com/
+      img: nexmo.png
+      tfa: Yes
+      sms: Yes
```

Comment:
@AlessioCasco From the screenshot, it looks like a phone call is an available method of 2FA. Can you please add the line `phone: Yes` if this is the case?